dashboard/views_test.py

```diff
@@ -175,6 +177,22 @@ def test_with_runs(self, mocked_refresh):
                     if course_data['runs'][0]['course_id'] == "course-v1:MITx+8.MechCX+2014_T1":
                         assert course_data['runs'][0]['status'] == CourseStatus.UPGRADE
 
+    @patch('backends.utils.refresh_user_token', autospec=True)
+    def test_certificates_enrollments(self, mocked_refresh):
+        """
+        Make sure we call the proper api functions
+        """
+        certificates = Certificates([])
+        enrollments = Enrollments([])
+
+        with patch('dashboard.views.get_student_certificates', autospec=True, return_value=certificates) as cert:
+            with patch('dashboard.views.get_student_enrollments', autospec=True, return_value=enrollments) as enroll:
```

Comment:
actually I was wrong: the context managers are one of the few cases the `\` is the only option...